pyproject.toml

```diff
@@ -36,7 +36,8 @@ dependencies = [
     "cryptography",  # MySQL & MariaDB dependency
     "pyodbc", # Microsoft SQL Server
     # Normalization
-    "phonenumbers"
+    "phonenumbers",
+    "lefthook"
```

Comment:
@johanna-skylight would you mind putting this after line 61 instead.  This is only necessary for developers and we won't need to install this in the upcoming AWS env running the demo.